tool/planet/start.go

```diff
@@ -161,6 +161,8 @@ func start(config *Config, monitorc chan<- bool) (*runtimeContext, error) {
 		box.EnvPair{Name: EnvElectionEnabled, Val: strconv.FormatBool(config.ElectionEnabled)},
 		box.EnvPair{Name: EnvDNSHosts, Val: config.DNS.Hosts.String()},
 		box.EnvPair{Name: EnvDNSZones, Val: config.DNS.Zones.String()},
+		box.EnvPair{Name: EnvServiceUID, Val: config.ServiceUser.UID},
+		box.EnvPair{Name: EnvServiceGID, Val: config.ServiceUser.GID},
```

Comment:
When do we want to add EnvPairs and how are they being used?

Reply:
@bernardjkim They are basically used to propagate information from the planet container's start configuration into the container itself - these variables are written to `/etc/container-environment` file inside the planet container and as such are made available to everything running inside planet - in this particular case these variables will be read/used by the planet agent.